bin/update-estimates.sh

```diff
@@ -8,16 +8,6 @@ rm last-update/update-complete
 declare -a targets=(
   "R/update-cases.R"
```

Comment:
Out of date due to the update-national.R change?

Reply:
The update-national.R is work in progress that got caught up by mistake - it's coming back out